<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>561</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Agency FB&quot;;
border:2px groove gray;border-radius:10px;padding:2px 4px;
background-image: url(:/resource/星空动态图.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/th (3).jpg);
background-image: url(:/resource/61V58PICDXw.jpg);
border-image: url(:/resource/按键6.png);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/61V58PICDXw.jpg);
border-image: url(:/resource/按键1副本.png);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/61V58PICDXw.jpg);
border-image: url(:/resource/按键5.png);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 28pt &quot;幼圆&quot;;
background-image: url(:/);
border:0px;</string>
    </property>
    <property name="text">
     <string>MyMusic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="about">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/);
font: 12pt &quot;方正舒体&quot;;
border:0px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>关于我们</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/按键14.png);
font: 9pt &quot;Agency FB&quot;;
background-image: url(:/resource/QQ图片20180719093059.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/QQ图片20180719093059.jpg);
border-image: url(:/resource/按键16.png);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(167, 255, 35);
border-image: url(:/);
background-image: url(:/);
border:0px;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>31</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/);
border:0px;
background-image: url(:/);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Agency FB&quot;;
background-image: url(:/);
border:0px;</string>
    </property>
    <property name="text">
     <string>               歌名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/按键13.png);
font: 9pt &quot;Agency FB&quot;;
background-image: url(:/resource/QQ图片20180719093059.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>361</width>
      <height>381</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;黑体&quot;;
background-image: url(:/);
border:0px;</string>
    </property>
    <property name="text">
     <string>歌单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>467</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;黑体&quot;;
background-image: url(:/);
border:0px;</string>
    </property>
    <property name="text">
     <string>添加音乐</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resource/图标音量.jpg);
background-image: url(:/resource/QQ图片20180719093059.jpg);
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/);
selection-background-color: rgb(0, 0, 0);
gridline-color: rgb(193, 215, 255);
background-image: url(:/);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/);
selection-background-color: rgb(0, 0, 0);
gridline-color: rgb(193, 215, 255);
background-image: url(:/);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_6">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>440</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;循环方式&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/61V58PICDXw.jpg);
border-image: url(:/resource/按键11.png);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <zorder>toolButton</zorder>
   <zorder>toolButton_2</zorder>
   <zorder>toolButton_3</zorder>
   <zorder>label</zorder>
   <zorder>about</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>toolButton_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>verticalSlider</zorder>
   <zorder>label_4</zorder>
   <zorder>label_2</zorder>
   <zorder>treeWidget</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>toolButton_5</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
   <zorder>toolButton_6</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>团队</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>关于播放器</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
